docs/running-on-yarn.md

```diff
@@ -492,7 +492,7 @@ for:
 
 If an application needs to interact with other secure Hadoop filesystems, their URIs need to be
 explicitly provided to Spark at launch time. This is done by listing them in the
-`spark.yarn.access.hadoopFileSystems` property, described in the configuration section below.
+`spark.kerberos.access.hadoopFileSystems` property, described in the configuration section below.
```

Comment:
Actually I was hoping to move this part to the `security.md` guide now, since it's not YARN-specific anymore.